.github/workflows/ci.yml

```
@@ -0,0 +1,46 @@
+name: Continuous integration
+
+on:
+  push:
+    branches-ignore:
+      - "master"
+      - "develop"
```

```suggestion
    branches-ignore:
      - "master"
      - "develop"
      - "release/**"
```

Other than in all other projects, release branches are not ignored by the ci pipeline. Is this intended? If yes: Should we discuss to run ci build on release branches as well?

---

There was no specific reason. I copied this from `sakuli-dashboard`. I think it's the only one where the release branch is not ignored :see_no_evil: